pkgs/development/libraries/AntTweakBar/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchurl, unzip, xlibs, mesa }:
+
+stdenv.mkDerivation rec {
+  name = "AntTweakBar-1.16";
+
+  buildInputs = [ unzip xlibs.libX11 mesa ];
+
+  src = fetchurl {
+    url = "https://vorboss.dl.sourceforge.net/project/anttweakbar/AntTweakBar_116.zip";
+    sha256 = "0z3frxpzf54cjs07m6kg09p7nljhr7140f4pznwi7srwq4cvgkpv";
+  };
+
+  postPatch = "cd src";
+  installPhase = ''
+    mkdir -p $out/lib/
+    cp -vi ../lib/{libAntTweakBar.so,libAntTweakBar.so.1,libAntTweakBar.a} $out/lib/
```

Comment:
Any reason for `-vi`? Verbose copying is generally not used in Nixpkgs for installation; and you are copying to an empty directory anyway.

Reply:
No reason. I was helped by others to create the nix expression and I didn't think it through enough. Will remove.